app/src/main/java/org/p2p/wallet/swap/ui/jupiter/main/JupiterSwapFragment.kt

```diff
@@ -1,63 +1,70 @@
 package org.p2p.wallet.swap.ui.jupiter.main
 
-import androidx.core.view.isVisible
+import androidx.core.view.isInvisible
 import android.os.Bundle
 import android.view.View
-import org.p2p.core.common.TextContainer
 import org.p2p.uikit.utils.drawable.DrawableCellModel
 import org.p2p.uikit.utils.drawable.applyBackground
 import org.p2p.uikit.utils.drawable.shape.rippleForeground
 import org.p2p.uikit.utils.drawable.shape.shapeCircle
 import org.p2p.uikit.utils.drawable.shape.shapeRoundedAll
 import org.p2p.uikit.utils.drawable.shapeDrawable
-import org.p2p.uikit.utils.skeleton.SkeletonCellModel
 import org.p2p.uikit.utils.text.TextViewCellModel
-import org.p2p.uikit.utils.text.bindSkeleton
+import org.p2p.uikit.utils.text.bind
 import org.p2p.uikit.utils.toPx
 import org.p2p.wallet.R
-import org.p2p.wallet.common.mvp.BaseFragment
+import org.p2p.wallet.common.mvp.BaseMvpFragment
 import org.p2p.wallet.databinding.FragmentJupiterSwapBinding
 import org.p2p.wallet.swap.ui.jupiter.main.widget.SwapWidgetModel
 import org.p2p.wallet.utils.viewbinding.viewBinding
 
-class JupiterSwapFragment : BaseFragment(R.layout.fragment_jupiter_swap) {
+class JupiterSwapFragment :
+    BaseMvpFragment<JupiterSwapContract.View, JupiterSwapContract.Presenter>(R.layout.fragment_jupiter_swap),
+    JupiterSwapContract.View {
 
     private val binding: FragmentJupiterSwapBinding by viewBinding()
+    override val presenter: JupiterSwapContract.Presenter
+        get() = TODO("Not yet implemented")
```

Comment:
TODO() is expected for error or replace with // TODO

Reply:
next pr will fixit it